recipes/python-3.5/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: python
+  version: 3.5.1
+
+source:
+  fn: Python-3.5.1.tgz
+  url: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
+  md5: be78e48cdfc1a7ad90efff146dce6cfe
+  patches:
+    - osx-dist.patch       # [osx]
+    - win-find_exe.patch   # [win]
+
+build:
+  no_link: 
+    - bin/python
+  number: 0
+
+requirements:
+  build:
+    - bzip2     # [unix]
+    - zlib      # [unix]
+    - sqlite    # [unix]
+    - readline  # [unix]
+    - tk        # [unix]
+    - openssl   # [unix]
+    - xz        # [unix]
+    - ncurses   # [linux]
```

Comment:
SVN is only required for windows.  It is used to download zlib, tk, and other libraries during the build. I don't think this can be installed through conda but it could be listed as somehow as a dependency?  SVN is available on AppVeyor but would be needed to installed on a users machine is they wanted to do the build on a local machine.